internal/telemetry/sensitivemetadataallowlist/sensitivemetadataallowlist.go

```diff
@@ -37,39 +37,123 @@ func AllowedEventTypes() EventTypes {
 				"testField",
 			},
 		},
-		// The 'languageId' key is included for feature:'cody.completions' action:suggested/accepted events to provide
+		// The 'languageId' key is included for feature:'cody.completions' events to provide
 		// customers with valuable language-specific insights from the analytics we offer.
 		// This information helps them better understand code completion usage patterns.
 		EventType{
 			Feature: "cody.completion",
-			Action:  "suggested",
+			Action:  "%",
```

Comment:
Conventional wildcard format is `*`:

```suggestion
			Action:  "*",
```

Reply:
ahh I was thinking with my SQL brain. Updated, thank you!